components/registry/org.wso2.carbon.registry.extensions/src/main/java/org/wso2/carbon/registry/extensions/handlers/utils/EndpointUtils.java

```diff
@@ -114,6 +121,12 @@ public static void saveEndpointsFromWSDL(String wsdlPath, Resource wsdlResource,
             log.error(msg, e);
```

Comment:
Logging and throwing both done is unnecessary. I think just throwing the error with the message is sufficient.